launchd/keybase.keybot.promote.darwin.plist

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>keybase.keybot.promote.darwin</string>
```

Comment:
i don't think you need darwin here cause launchd plists are only darwin?

When changing here be sure to update log files.